fetcher-pipeline/file_utils.py

```diff
@@ -125,3 +127,10 @@ def write_hash(file: Path):
     hash_file = file.parent.joinpath(".hash")
     with hash_file.open('w') as f:
         f.write(str(hash_val))
+
+def calc_md5_hash(file_path) -> str:
+    try:
+        with open(file_path, "rb") as file, mmap(file.fileno(), 0, access=ACCESS_READ) as file:
```

Comment:
Interesting use of `mmap()` here.  Is there a specific reason to memory-map the file when calculating the hash?

Reply:
This is 100% an @mXaln  question. He wrote this for localdev, and it was calculating proper md5 hashes, so I just ran with it. Figured he knew. But a shote in the dark says maybe in the  case of any potentially pretty large wav files you avoid RAM overhead of reading it all at once.